server/webapp/localize.xml

```diff
@@ -794,4 +794,8 @@
     <entry key="CAN_NOT_UPDATE_RESOURCES_ON_ELASTIC_AGENT">Resources on elastic agents with uuids [{0}] can not be updated.</entry>
     <entry key="PENDING_AGENT_INVALID_OPERATION">Pending agents [{0}] must be explicitly enabled or disabled when performing any operations on them</entry>
     <entry key="NO_OPERATION_PERFORMED_ON_AGENTS">No Operation performed on agents.</entry>
+    <entry key="PIPELINE_PAUSE_SUCCESSFUL">Pipeline ''{0}'' paused successfully.</entry>
```

Comment:
`localize_ja.xml` hasn't been deleted, so, you need to add these entries there too?

Reply:
Shouldn't we delete `localize_ja.xml`?

Reply:
Yes. I remember @GaneshSPatil and me talking about it. I suppose another PR for it? @ketan wanted to get rid of even localize.xml, I think

Reply:
> Shouldn't we delete localize_ja.xml?

I vote for you for president...